bedrock/base/templates/includes/protocol/footer/footer-refresh.html

```diff
@@ -66,7 +66,7 @@ <h2 class="moz24-footer-label" data-testid="footer-heading-developers">
             <div class="moz24-footer-refresh-social-wrapper">
               <h2 class="moz24-footer-heading-social">{{ ftl('footer-refresh-follow-mozilla') }}</h2>
               <ul class="moz24-footer-links-social">
-                <li><a class="twitter" href="{{ mozilla_twitter_url() }}" data-link-position="footer" data-link-text="Twitter (@mozilla)">{{ ftl('footer-refresh-x-formerly-twitter', fallback='footer-twitter') }}<span> (@mozilla)</span></a></li>
```

Comment:
Can both the helpers https://github.com/mozilla/bedrock/blob/6aafa46ea64613f1f74da27346e277de6e208117/bedrock/mozorg/templatetags/misc.py#L420-L425 be removed now, incl. tests?

Reply:
yes, I was checking if we wanted to repurpose for Bluesky, but will remove as it seems we don't have localised account there yet